api/service/version_endpoint_service.go

```diff
@@ -172,6 +172,32 @@ func (k *endpointService) override(left *models.VersionEndpoint, right *models.V
 		left.ResourceRequest = right.ResourceRequest
 	}
 
+	// override logger
+	if right.Logger != nil {
+		left.Logger = right.Logger
+		left.Logger.DestinationURL = k.loggerDestinationURL
+		predictionLogger := left.Logger.Prediction
+		if predictionLogger != nil && right.Protocol == protocol.UpiV1 {
+			left.Logger.Prediction = predictionLogger
+		}
```

Comment:
I don't see a value in having these codes. There's no change in predictionLogger object

Reply:
nice catch! Updated